frontend/src/pages/Admin/UserInfo.js

```diff
@@ -1,19 +1,22 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
   Label, TextInput, Grid, Fieldset,
 } from '@trussworks/react-uswds';
 import moment from 'moment';
-
 import RegionDropdown from '../../components/RegionDropdown';
-import JobTitleDropdown from '../../components/JobTitleDropdown';
```

Comment:
Can we delete this component? It appears to be unused now.

Reply:
Ah yes, I meant to remove it, but then it never happened. Thanks for catching it.